ProcessLib/LIE/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -282,6 +284,30 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
                 "time-dependent.",
                 frac_prop->aperture0.name.c_str());
         }
+
+        auto permeability_model_config =
```

Comment:
As that in my comments in #2542, a creator function is better for this part because this part is for the material property and it can be process independent.

Reply:
The function can be put in MaterialLib/FractureModels/